sdk/communication/Azure.Communication.CallingServer/tests/SessionRecords/CallMediaLiveTests/PlayAudioAsync.json

```diff
@@ -0,0 +1,178 @@
+            "Content-Type": "application/json",
+            "User-Agent": "azsdk-net-Communication.CallingServer/1.0.0-alpha.20220809.1 (.NET Framework 4.8.4510.0; Microsoft Windows 10.0.19044 )",
+            "x-ms-client-request-id": "e582e407340285d11f289281a4fae38a",
+            "x-ms-content-sha256": "Sanitized",
+            "x-ms-date": "Tue, 09 Aug 2022 23:10:44 GMT",
+            "x-ms-return-client-request-id": "true"
+        },
+        "RequestBody": {
+            "playSourceInfo": {
+                "sourceType": "file",
+                "playSourceId": "playSourceId",
+                "fileSource": {
+                    "uri": "https://localhost/bot-hold-music-2.wav"
+                }
+            },
+            "playTo": [
+                {
+                    "rawId": "Sanitized",
+                    "communicationUser": {
+                        "id": "Sanitized"
+                    }
+                }
+            ]
+        },
+        "StatusCode": 202,
+        "ResponseHeaders": {
+            "Content-Length": "0",
+            "Date": "Tue, 09 Aug 2022 23:10:46 GMT",
+            "X-Azure-Ref": "0dOnyYgAAAACxr\u002BFfOF6nSbaw\u002BReuTM7pV1NURURHRTA4MTUAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+            "X-Cache": "CONFIG_NOCACHE",
+            "X-Microsoft-Skype-Chain-ID": "be26aae5-de21-446c-aae0-1bfca0ed7dc6",
+            "x-ms-client-request-id": "e582e407340285d11f289281a4fae38a"
+        },
+        "ResponseBody": null
+    },
+    {
+      "RequestUri": "https://sanitized.skype.com/calling/callConnections/401f0b00-13d4-43f9-9c34-01b56eb262e3:terminate?api-version=2022-04-07-preview",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Authorization": "Sanitized",
+        "Content-Length": "0",
+        "User-Agent": "azsdk-net-Communication.CallingServer/1.0.0-alpha.20220809.1 (.NET Framework 4.8.4510.0; Microsoft Windows 10.0.19044 )",
+        "x-ms-client-request-id": "7688a55ee1787d100a45edda2dd31b2f",
+        "x-ms-content-sha256": "Sanitized",
+        "x-ms-date": "Tue, 09 Aug 2022 23:10:47 GMT",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 204,
+      "ResponseHeaders": {
+        "Date": "Tue, 09 Aug 2022 23:10:46 GMT",
+        "X-Azure-Ref": "0d\u002BnyYgAAAADgNS\u002BVilD\u002BQorCkB5iYOE6V1NURURHRTA4MTUAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "X-Microsoft-Skype-Chain-ID": "be26aae5-de21-446c-aae0-1bfca0ed7dc6",
+        "x-ms-client-request-id": "7688a55ee1787d100a45edda2dd31b2f"
+      },
+      "ResponseBody": null
+    }
+  ],
+  "Variables": {
+    "RandomSeed": "1352044465",
+    "COMMUNICATION_LIVETEST_STATIC_CONNECTION_STRING": "endpoint=https://acs-recording-common-e2e.communication.azure.com/;accesskey=Kg==",
+    "TARGET_USER_ID": "8:acs:70559d7c-67b2-444f-a1f3-8027a1412880_00000013-24bb-ba02-d68a-084822008625",
```

Comment:
do we need to sanitize the resource id or is it just a random one?

Reply:
you mean the ConnectionString or the Target User ID?

Reply:
TARGET_USER_ID

Reply:
good catch, thank you. I replaced it with a random GUID.